config/tutorials/otel-logs/index-config.yaml

```diff
@@ -5,30 +5,72 @@
 
 version: 0.6
 
-index_id: otel-log-v0
+index_id: otel-logs-v0_6
```

Comment:
I updated them here because these configs seem more like a reference to me. we should keep them up to date whether they are used in a tut or not. 
I can move the change to the other PR if it makes more sense.